Dockerfile.production

```diff
@@ -0,0 +1,75 @@
+FROM ruby:3.3.0
+
+# Install essential packages
+RUN apt-get update -yq \
+    && apt-get install -y -qq --no-install-recommends \
+        build-essential \
+        curl \
+        git-core \
+        vim \
+        passwd \
+        unzip \
+        cron \
+        gcc \
+        wget \
+        netcat-traditional \
+    && apt-get clean
+
+# Install Node.js 18 and Yarn
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
+    && apt-get install -y --no-install-recommends nodejs \
+    && npm install -g yarn \
+    && apt-get clean
+
+# Install other required packages
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        imagemagick \
+        postgresql-client \
+        libvips \
+    && apt-get clean
+
+# Set up the app directory
+RUN mkdir /app
+WORKDIR /app
+
+# Copy package.json and yarn.lock
+COPY package.json yarn.lock ./
+
+# Install Node.js dependencies with Yarn
+RUN yarn install
+
+# Install specific version of bundler
+RUN gem install bundler -v 2.2.32
+
+# Set environment variables
+ENV BUNDLE_GEMFILE=/app/Gemfile \
+    BUNDLE_JOBS=20 \
+    BUNDLE_PATH=/bundle \
+    BUNDLE_BIN=/bundle/bin \
+    GEM_HOME=/bundle \
+    RAILS_ENV=production \
+    RAILS_SERVE_STATIC_FILES=1 \
+    RAILS_LOG_TO_STDOUT=1
+
+ENV PATH="${BUNDLE_BIN}:${PATH}"
+
+# Copy Gemfile and Gemfile.lock for Ruby dependencies
+COPY Gemfile Gemfile.lock ./
+RUN bundle install --binstubs --without development test
+
+# Copy the rest of the application code
+COPY . .
+
+# Precompile assets
+RUN SECRET_KEY_BASE=skb DB_ADAPTER=nulldb bundle exec rails assets:precompile
+
+# Set execute permission for entrypoint script
+# RUN chmod +x entrypoint.sh
+# ENTRYPOINT ["./entrypoint.sh"]
+
+# Expose port 4000
+EXPOSE 4000
+
+# Default command to run the container
+CMD ["bash"]
```

Comment:
this dockerfile does not look like the one from rails 8 to me..

production would have thruster running